transport/transport.go

```diff
@@ -185,13 +185,17 @@ type Stream struct {
 
 	headerChan chan struct{} // closed to indicate the end of header metadata.
 	headerDone uint32        // set when headerChan is closed. Used to avoid closing headerChan multiple times.
-	header     metadata.MD   // the received header metadata.
-	trailer    metadata.MD   // the key-value map of trailer metadata.
+	// hdrMu protects header and trailer metadata on the server-side.
```

Comment:
Please add a blank line above this to set it off from the above fields.

Reply:
done.